addons/im_livechat/controllers/channel.py

```diff
@@ -0,0 +1,26 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.http import request, route
+from odoo.addons.mail.controllers.discuss.channel import ChannelController
+
+
+class ChannelControllerLiveChat(ChannelController):
+    @route()
+    def discuss_channel_info(self, channel_id):
+        res = super().discuss_channel_info(channel_id)
+        if not res and request.env.user.has_group("im_livechat.im_livechat_group_manager"):
+            channel = request.env["discuss.channel"].sudo().browse(channel_id)
+            channel_info = channel._channel_info()
```

Comment:
It's a bit unfortunate that the "logic" of the base method is duplicated on the overrides.
Maybe we should provide better extension points.
But as the code is so small, maybe it's not worth it.